LabeledIconButton/LabeledIconButton.js

```diff
@@ -83,6 +83,14 @@ const LabeledIconButtonBase = kind({
 		 */
 		iconComponent: EnactPropTypes.component,
 
+		/**
+		 * True if button is an icon only button.
+		 *
+		 * @type {Boolean}
+		 * @private
+		 */
+		iconOnly: PropTypes.bool,
```

Comment:
I don't know why we should add prop here. Do we have to add this for samplers?

Reply:
As I was saying, to fix the warning, I need `delete rest.iconOnly;` line and it needs this propTypes for lint warning.